Cargo.toml

```diff
@@ -23,6 +23,9 @@ lazy_static = "1.4.0"
 pest = "2.0"
 pest_derive = "2.0"
 itertools = "0.10.5"
+tempdir = "0.3.7"
+tempfile = "3.6.0"
+tar = "0.4.38"
```

Comment:
tar is just a development dependency though? can these be separated? actually - all three of these are just used in tests